File: Iniciante/1065.c
#include <stdio.h>
 
int main() {

    int i, n, contN=0;

    for(i=0; i < 5; i++){
        scanf("%d", &n);
        if(n % 2 == 0){
            contN++;
        }
    }
        printf("%d valores pares\n", contN);

    return 0;
}

File: Iniciante/1065.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {

        Scanner scanf = new Scanner(System.in);

        int contN=0;

        for(int i=0; i < 5; i++){
            int n = scanf.nextInt();
            if(n % 2 == 0){
                contN++;
            }
        }
            System.out.printf("%d valores pares\n", contN);
    }
}